webofneeds/won-owner-webapp/src/main/webapp/config/usecases/classifieds.js

```diff
@@ -0,0 +1,243 @@
+/**
+ * Created by kweinberger on 06.12.2018.
+ */
+
+import { details, emptyDraft } from "../detail-definitions.js";
+import won from "../../app/won-es6.js";
+import { getIn, is } from "../../app/utils.js";
+import {
+  // filterInVicinity,
+  textSearchSubQuery,
+  vicinityScoreSubQuery,
+  filterRentRange,
+  concatenateFilters,
+  sparqlQuery,
+} from "../../app/sparql-builder-utils.js";
+
+/*
+TODO: send hints to counterparts of searches (offers)
+TODO: create use case icons
+*/
+
+const classifiedsUseCases = {
+  goodsOffer: {
+    identifier: "goodsOffer",
+    label: "Offer Something",
+    icon: "#ico36_plus",
+    draft: {
+      ...emptyDraft,
+      content: {
+        type: "s:Offer",
+      },
+      seeks: {
+        type: "s:Demand",
+      },
+    },
+    details: {
+      title: { ...details.title, mandatory: true },
+      description: { ...details.description },
+      price: { ...details.price, mandatory: true },
+      tags: { ...details.tags },
+      location: { ...details.location },
+      images: { ...details.images },
+    },
+    // TODO: what should this match on?
+    // generateQuery: (draft, resultName) => {
+    //   // return query
+    // },
+  },
+  goodsDemand: {
+    identifier: "goodsDemand",
+    label: "Look for something",
+    icon: "#ico36_plus",
+    draft: {
+      ...emptyDraft,
+      content: {
+        title: "I'm looking for something!",
```

Comment:
should not be set, default titles could be handled acc. to needtype on viewer/generateHumanReadable